### PR TITLE
[SDC-9317] - added delete bin functionality

### PR DESCRIPTION
**Pull request by zlosim**
Hi @jarcec , 
I manged to mess previous pull request https://github.com/streamsets/datacollector/pull/68 so had to create new one, 
sorry for the delay, but finally had time to implement CDC style delete operation for aerospike

let me know if this is good enough

### CODE REVIEW

File: aerospike-lib/src/main/java/com/streamsets/pipeline/destination/aerospike/AerospikeErrors.java [hunk 1]
```diff
@@ -26,7 +26,8 @@
   AEROSPIKE_04("Put operation was not successful '{}'"),
   AEROSPIKE_05("There has to be at least one bin to store"),
   AEROSPIKE_06("'{}' can not be parsed: '{}'"),
-  AEROSPIKE_07("'{}' can not be evaluated: '{}'"),;
+  AEROSPIKE_07("'{}' can not be evaluated: '{}'"),
+  AEROSPIKE_08("'{}' is not supported operation");
```
Comment (jarcec) [inline aerospike-lib/src/main/java/com/streamsets/pipeline/destination/aerospike/AerospikeErrors.java]: Nit: let's put the semicolon on its own line and then end this lien with a comma. The idea is that next time we need to add a new error code, we only add a new line and we don't have to change any existing lines - which makes using git blame or git cherrypick much much easier.

File: aerospike-lib/src/main/java/com/streamsets/pipeline/destination/aerospike/AerospikeOperationType.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+package com.streamsets.pipeline.destination.aerospike;
+
+import com.streamsets.pipeline.api.Label;
+import com.streamsets.pipeline.api.impl.Utils;
+import com.streamsets.pipeline.lib.operation.OperationType;
+
+public enum AerospikeOperationType implements Label {
+  DELETE(OperationType.DELETE_CODE),
+  UPSERT(OperationType.UPSERT_CODE),
+      ;
```
Comment (jarcec) [inline aerospike-lib/src/main/java/com/streamsets/pipeline/destination/aerospike/AerospikeOperationType.java]: A bit off indentation.

File: aerospike-lib/src/main/java/com/streamsets/pipeline/destination/aerospike/AerospikeOperationType.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2017 StreamSets Inc.
```
Comment (jarcec) [inline aerospike-lib/src/main/java/com/streamsets/pipeline/destination/aerospike/AerospikeOperationType.java]: Do you mind updating all the files to have the copyright year set in 2019?

File: aerospike-lib/src/main/java/com/streamsets/pipeline/destination/aerospike/AerospikeErrors.java (hunk 1 above)
Comment (zlosim) [reply aerospike-lib/src/main/java/com/streamsets/pipeline/destination/aerospike/AerospikeErrors.java]: thanks for the tip! done

File: aerospike-lib/src/main/java/com/streamsets/pipeline/destination/aerospike/AerospikeOperationType.java (hunk 2 above)
Comment (zlosim) [reply aerospike-lib/src/main/java/com/streamsets/pipeline/destination/aerospike/AerospikeOperationType.java]: fixed

File: aerospike-lib/src/main/java/com/streamsets/pipeline/destination/aerospike/AerospikeOperationType.java (hunk 3 above)
Comment (zlosim) [reply aerospike-lib/src/main/java/com/streamsets/pipeline/destination/aerospike/AerospikeOperationType.java]: @jarcec should be done :)

**Comment by jarcec** [comment]
Thanks, @zlosim , let me try to push the patch through our Jenkins.

**Comment by jarcec** [comment]
I've just merged the patch, thank you for the contribution @zlosim !

**Pull request closed without merging**